pkg/ddc/jindo/operations/base.go

```diff
@@ -107,3 +107,40 @@ func (a JindoFileUtils) GetUfsTotalSize(url string, useStsSecret bool) (summary
 	}
 	return stdout, err
 }
+
+// Check if the Alluxio is ready by running `alluxio fsadmin report` command
+func (a JindoFileUtils) Ready() (ready bool) {
+	var (
+		command = []string{"/sdk/bin/jindo", "jfs", "-report"}
```

Comment:
hard code

Reply:
thx,this is jindofs's fixed shell command, so be hard code

Reply:
You can define constant in common pkg.

Reply:
just same as with other many command , will replace "/sdk/bin/jindo" by "jindo"